### ISSUE TITLE
Splunk management server support

### ISSUE THREAD

**Issue by badarsebard** [splunk/terraform-provider-splunk#6]
This might be tricky because it's not doable through the API, but it would be nice to be able to set configurations for Splunk management servers (deployer, cluster master, deployment server). Most of this can be achieved through the configs and provider aliasing, but there are other aspects that can't such as putting apps to be deployed into their staging directories (i.e. shcluster/apps, deployment-apps, master-apps). In addition, it would be nice to have documentation around common Splunk commands for these servers such as rolling restarts and `reload deploy-server`.

**Comment by anushjay** [comment]
@badarsebard Agree. We focused on resource creation in this first release, we are now looking into basic commands such as `restart`, etc. We will add this to our list of things to do. Thanks for your suggestion again.
